project/AppDependencies.scala

```diff
@@ -6,20 +6,20 @@ object AppDependencies {
 
   val appName = "income-tax-subscription-self-employed-frontend"
 
-  private val playLanguageVersion = "5.2.0-play-28"
-  private val bootstrapVersion = "5.24.0"
-  private val playHmrcFrontendVersion = "3.8.0-play-28"
+  private val playLanguageVersion = "5.3.0-play-28"
+  private val bootstrapVersion = "7.4.0"
```

Comment:
The latest version is:
```suggestion
  private val bootstrapVersion = "7.6.0"
```